includes/class-sensei-posttypes.php

```diff
@@ -605,11 +607,13 @@ public function setup_question_category_taxonomy() {
 			'all_items'         => __( 'All Question Categories', 'sensei-lms' ),
 			'parent_item'       => __( 'Parent Question Category', 'sensei-lms' ),
 			'parent_item_colon' => __( 'Parent Question Category:', 'sensei-lms' ),
+			'view_item'         => __( 'View Course Category', 'sensei-lms' ),
```

Comment:
View Question Category

Reply:
@donnapep Good catch. I fixed it, but we probably could also leave this one out, since it doesn't seem to be shown in the admin-page.